docs/execution/execute.rst

```diff
@@ -40,22 +41,23 @@ You can pass variables to a query via ``variables``.
 
 .. code:: python
 
-    class Query(graphene.ObjectType):
-        user = graphene.Field(User, id=graphene.ID(required=True))
+    class Query(ObjectType):
```

Comment:
Whenever I read docs examples like this I always ask "where the hell do I import that from though?".

I feel like if we remove the `graphene` bit we need to show people where it is imported from. Because copy/pasting this won't work.

Reply:
This is something I've been meaning to bring up. I think we should, wherever possible, make snippets in the docs complete. That is, again as much as possible, allow them to be copy/pasted into a `.py` file and actually work. I echo your thought here.

Reply:
+1 on including imports. I always get confused with that too.

Reply:
Also importing a package would be a better approach rather than import each class separately. Makes reading code much easier if each external reference is easily identifiable.